mock-backend/src/controllers/productController.js

```diff
@@ -1,3 +1,5 @@
+const {categoryFilter} = require('../utils/categoryFilter')
```

Comment:
This lackf formatting. Should be like `{ categoryFilter }`

Reply:
Done